webapp/components/test-search.js

```diff
@@ -487,8 +487,7 @@ class TestSearch extends WPTFlags(PolymerElement) {
       let matches = Array.from(paths);
       if (query) {
         matches = matches
-          .filter(p => p.toLowerCase())
-          .filter(p => p.includes(query))
+          .filter(p => p.toLowerCase().includes(query))
```

Comment:
Good catch! The original version definitely couldn't be the intended behavior, as it arbitrarily converted the strings to lowercase and only filtered empty strings.